src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/xlf/PackageReference.xaml.de.xlf

```diff
@@ -77,16 +77,6 @@
         <target state="translated">Speicherort des Pakets, auf das verwiesen wird.</target>
         <note />
       </trans-unit>
-      <trans-unit id="BoolProperty|GeneratePathProperty|DisplayName">
-        <source>Generate path property</source>
-        <target state="new">Generate path property</target>
-        <note />
-      </trans-unit>
```

Comment:
This part of file was updated between moment we took xlf and localized id till moment will created PR.
Please merge file.

Reply:
@jmarolf this PR updated xlf file https://github.com/dotnet/project-system/commit/df5840152cca05183bff5d5d10c060a081ed05c2#diff-c550a87f8230c13f67d13c3ddfa142bb